src/PrestaShopBundle/Resources/config/routing/admin/improve/design/theme.yml

```diff
@@ -12,27 +12,33 @@ admin_themes_upload_logos:
   defaults:
     _controller: PrestaShopBundle:Admin\Improve\Design\Theme:uploadLogos
     _legacy_controller: AdminThemes
+    _legacy_link: AdminThemes:submitOptionsconfiguration
```

Comment:
Sure about this one ? `Theme:uploadLogos` and `AdminThemes:submitOptionsconfiguration` seem very different but I trust you 😄

Reply:
yes it truly is :D 
![Screenshot from 2019-06-05 10-25-40](https://user-images.githubusercontent.com/17051250/58937905-67485980-877c-11e9-87a2-f71d506eb71d.png)